spring-boot-starter/example-web/README.md

```diff
@@ -38,7 +38,7 @@ public class RestApplication {
 
 3. You can also put BPMN, CMMN and DMN files in your classpath, they will be automatically deployed and registered within a process application.
 
->Note: Basic authentication is configured on Spring Boot level, this is NOT a Camunda Engine authentication. Check [this docs](https://docs.camunda.org/manual/7.13/reference/rest/overview/authentication/) 
+>Note: Basic authentication is configured on Spring Boot level, this is NOT a Camunda Engine authentication. Check [this docs](https://docs.camunda.org/manual/7.14/reference/rest/overview/authentication/) 
```

Comment:
Should we simply go for `latest` here so it doesn't have to be upgraded every time?